code/modules/clothing/outfits/standard.dm

```diff
@@ -400,11 +400,8 @@
 	name = "Timeline Eradication Agent"
 
 	uniform = /obj/item/clothing/under/color/white
-	suit = /obj/item/clothing/suit/space/chronos
 	suit_store = /obj/item/tank/internals/oxygen
-	back = /obj/item/chrono_eraser
-	head = /obj/item/clothing/head/helmet/space/chronos
-	mask = /obj/item/clothing/mask/breath
```

Comment:
Wont they still need a breathmask?

Reply:
Ideally they'd just breathe through the modsuit but yeah I'll readd it